src/dune_rules/melange_stanzas.ml

```diff
@@ -47,6 +48,15 @@ module Emit = struct
     t
 
   let decode =
+    let extension_field name =
```

Comment:
This function could prob be reused in `extension` field in the dialects stanza. I tried adding it to `Stanza_common` but the dependencies are the other way around (`dune_rules` depends on `dune_engine` where dialects stanza lives).